src/services/music/MusicUtils.ts

```
@@ -0,0 +1,26 @@
+import { Client, GuildMember } from 'discord.js';
+const ftd = 1;
```

ESLint

[error] 'ftd' is assigned a value but never used.

(no-unused-vars)